documentation/how-to-topics/Wcf-service-with-VBnet.md

```diff
@@ -0,0 +1,173 @@
+# Using WCF with VB.Net
+
+## Introduction
+
+VB.Net is only supported in OpenSilver 2.0+ (including 2.0 preview). It is not supported by the earlier versions.
+
+OpenSilver provides support for web services and HTTP calls in multiple ways, including:
+
+* The *"Add Service Reference"* feature of Visual Studio 2022. Use this feature to implement SOAP based WCF service however *"Add Service Reference"* feature tool is not available for VB.Net projects.
+> :warning: **Important points to notice **:
+> * When adding a WCF Service Reference, please be sure to uncheck the option "Reuse types in referenced assemblies".
+> * Update the `System.ServiceModel.*` NuGet packages from v4.4 to v4.10.
+> * Some code like event handlers may not be generated, in this case include the Silverlight version of Reference.cs file into the project.
+> * Binary serialization is not yet fully supported, please adjust new endpoint with `basicHttpBinding` in the `web.config`.
+ 
+**Note**: to enable passing cookies (for credentials/authentication), put the following code in your App.xaml.cs constructor:
+```
```

Comment:
After the opening triple quotes you can specify the language, then you will get a better syntax highlighting. Some examples:

**BEFORE:**
```
if (something is null)
{
    var test = "some string";
}
```
**AFTER:**
```c#
if (something is null)
{
    var test = "some string";
}
```

**BEFORE:**
```
If something is Nothing Then
    Dim test = "some string"
End If
```
**AFTER:**
```vb
If something is Nothing Then
    Dim test = "some string"
End If
```